.goreleaser.yml

```diff
@@ -5,6 +5,8 @@ before:
 build:
   binary: revealgo
   main: cmd/revealgo/reveal.go
+  ldflags:
+  - -s -w -X github.com/yusukebe/revealgo.Version=v{{.Version}}
```

Comment:
As you can see, now we are passing the version value as build flag. Meaning that we *must* create a git tag per each release otherwise it will not work as expected.

Reply:
Also see the change on version.go to make this work